velox/exec/tests/PlanNodeToStringTest.cpp

```diff
@@ -249,6 +249,17 @@ TEST_F(PlanNodeToStringTest, aggregation) {
   ASSERT_EQ(
       "-- Aggregation[SINGLE [c0] a0 := array_agg(ROW[\"c1\"]) ORDER BY c2 DESC NULLS LAST] -> c0:BIGINT, a0:ARRAY<INTEGER>\n",
       plan->toString(true, false));
+
+  // Aggregation over GroupId with global grouping sets.
+  plan = PlanBuilder()
+             .values({data_})
+             .groupId({"c0"}, {{"c0"}, {}}, {"c1"})
+             .singleAggregation({"c0", "group_id"}, {"sum(c1) as sum_c1"}, {})
+             .planNode();
+  ASSERT_EQ("-- Aggregation\n", plan->toString());
+  ASSERT_EQ(
+      "-- Aggregation[SINGLE [c0, group_id] sum_c1 := sum(ROW[\"c1\"]) global group IDs: [ 1 ] Group Id column: group_id] -> c0:SMALLINT, group_id:BIGINT, sum_c1:BIGINT\n",
```

Comment:
Would you add another case with > 1 grouping set?

Reply:
@mbasmanova : Bit confused about this one. The GroupId has 2 grouping sets {c0} and {} which makes the grouping keys at the aggregationNode [c0, group_id]. Please can you elaborate.

Reply:
@aditi-pandit Sorry, I wasn't clear. I wanted to ask to add a case of 2+ global grouping sets.

Reply:
Got it. Done.